trie/iterator.go

```diff
@@ -361,10 +361,16 @@ func (it *nodeIterator) resolveHash(hash hashNode, path []byte) (node, error) {
 			}
 		}
 	}
-	resolved, err := it.trie.resolveHash(hash, path)
-	return resolved, err
+	return it.trie.resolveHash(hash, path)
+}
+func (it *nodeIterator) resolveBlob(hash hashNode, path []byte) ([]byte, error) {
```

Comment:
When do we use this function?

Reply:
It's used by NodeBlob functionality, but seem we don't support it in the current logic, let me remove this.